src/lib.rs

```diff
@@ -29,11 +29,21 @@ use cosmwasm_vm::{
 };
 
 use crate::args::{
-    ARG1, ARG2, CACHE_ARG, CODE_ID_ARG, ENV_ARG, GAS_USED_ARG, INFO_ARG, MSG_ARG, WASM_ARG,
+    ARG1, ARG2, CACHE_ARG, CODE_ID_ARG, CONTRACT_INFO_ARG, ENV_ARG, GAS_USED_ARG, INFO_ARG,
+    MSG_ARG, WASM_ARG,
 };
 use crate::cache::{cache_t, to_cache};
 use crate::error::{handle_c_error, Error};
 
+#[derive(Copy, Clone)]
+#[repr(C)]
+/// A set of flags exposing some aspects of the contract configuration
+pub struct ContractFlags {
+    pub ibc_enabled: bool,
```

Comment:
I wonder if we find a better term for "enabled" here. We know nothing about the contract's ability to use IBC. It just has some exports that can even have a completely broken signature. So I would not derive any capability from that.

What about `wants_ibc`? It signals the intend to use IBC, we prepare some setup, the contract is charged for that.

Reply:
Naming is my weak spot.
@alpe What do you think of the names? `ibc_enabled`, `wants_ibc`, `handles_ibc_messages`, something else?